plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -109,6 +111,7 @@ public Composite createComposite(Composite parent, IWizardHandle handle) {
 		this.model = new ServerSettingsViewModel(server, connection);
 		DataBindingContext dbc = new DataBindingContext();
 		Composite composite = createControls(parent, model, dbc);
+		uiHook = composite.getChildren()[0];
```

Comment:
I will explain why uiHook is received this way.
First, it should not be parent composite, but a light control as Label, otherwize fragment will keep the entire tree of widgets, and the fragment is never disposed. 
Second, I did not assigned uiHook to a label where it is created because I recommend reuse methods creating ui widgets for this class and OpenShiftServerEditorFragment (see my comment to JBIDE-21410)